charts/kserve-resources/templates/clusterservingruntimes.yaml

```diff
@@ -30,59 +30,6 @@ spec:
           cpu: "1"
           memory: 2Gi
 
----
-apiVersion: serving.kserve.io/v1alpha1
-kind: ClusterServingRuntime
-metadata:
-  name: kserve-mlserver
```

Comment:
We are removing mlserver as well ?

Reply:
Yes, following your suggestion in https://github.com/kserve/kserve/issues/3380#issuecomment-1905199227

> Remove Seldon MLServer as we have equivalent features in KServe as well now, we already started removing MLServer reference in KServe docs.

Reply:
Should we keep it instead?